lib/builder-webpack4/src/index.ts

```diff
@@ -49,7 +49,7 @@ export const getConfig: WebpackBuilder['getConfig'] = async (options) => {
       ...options,
       babelOptions,
       typescriptOptions,
-      [`${options.framework}Options`]: frameworkOptions,
+      frameworkOptions: framework?.options,
```

Comment:
Need to:
- [ ] update `framework-preset-react` to use `frameworkOptions` instead of `reactOptions`
- [ ] update angular, etc.
- [ ] Add a MIGRATION.md note
- [ ] Update vite builder